tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -145,6 +146,51 @@ test.describe.serial(`AI Lab extension installation and verification`, { tag: '@
     });
   });
 
+  ['ggerganov/whisper.cpp'].forEach(modelName => {
+    test.describe.serial(`Model service creation and deletion`, () => {
+      let catalogPage: AILabCatalogPage;
+      let modelServiceDetailsPage: AILabServiceDetailsPage;
+
+      test.skip(isLinux, `Skipping model service creation on Linux`);
+      test.beforeAll(`Open AI Lab Catalog`, async ({ runner, page, navigationBar }) => {
+        [page, webview] = await handleWebview(runner, page, navigationBar);
+        aiLabPage = new AILabPage(page, webview);
+        await aiLabPage.navigationBar.waitForLoad();
+
+        catalogPage = await aiLabPage.navigationBar.openCatalog();
+        await catalogPage.waitForLoad();
+      });
+
+      test(`Download ${modelName} model if not available`, async () => {
+        test.setTimeout(310_000);
+        if (!(await catalogPage.isModelDownloaded(modelName))) {
+          await catalogPage.downloadModel(modelName);
+        }
+        await playExpect
+          // eslint-disable-next-line sonarjs/no-nested-functions
+          .poll(async () => await waitForCatalogModel(modelName), { timeout: 300_000, intervals: [5_000] })
+          .toBeTruthy();
+      });
+
+      test(`Create model service for ${modelName}`, async () => {
+        test.setTimeout(310_000);
+        const modelServiceCreationPage = await catalogPage.createModelService(modelName);
+        await modelServiceCreationPage.waitForLoad();
+
+        modelServiceDetailsPage = await modelServiceCreationPage.createService();
```

Comment:
It would be interesting to actually check for particular model in the ModelServiceCreationPage since it was called from catalog. We should see that model offered in the dropdown. In case when we have mutliple models available.

Reply:
When creating it from catalog that is not really required since the wanted model is autopopulated, you are desscribing a different test case altogether.